### ISSUE TITLE
Add user script not working

### ISSUE THREAD

**Issue by aitianshi** [uniuuu/zotprime#13]
When running the `bin/create-user.sh` script (i.e `bash create-user.sh 10000 aitianshi aitianshi`), I got the following error:
```
ERROR 1136 (21S01) at line 1: Column count doesn't match value count at row 1
ERROR 1136 (21S01) at line 1: Column count doesn't match value count at row 1
ERROR 1452 (23000) at line 1: Cannot add or update a child row: a foreign key constraint fails (`zotero_master`.`groupUsers`, CONSTRAINT `groupUsers_ibfk_2` FOREIGN KEY (`userID`) REFERENCES `users` (`userID`) ON DELETE CASCADE)
ERROR 1136 (21S01) at line 1: Column count doesn't match value count at row 1
```

I am gonna make a PR later (and link this issue) to fix it and improve the user script by adding some checks and logs.

**Comment by uniuuu** [comment]
Hi @aitianshi 
This issue has been fixed in [2.6.2](https://github.com/uniuuu/zotprime/releases/tag/v2.6.2)
Could you please test it?
To try the latest solution either remove repository and clone it again or change version in .env file to 2.6.2 and pull the images (alternatively build them with file docker-compose-dev.yml ).

**Comment by uniuuu** [comment]
Hi @aitianshi 
The new [v2.7.0](https://github.com/uniuuu/zotprime/releases/tag/v2.7.0) has been released with latest client 6.0.28 and latest dataserver code. 
Kindly delete your local copy of repository and clone it again. 
Please reopen this ticket if you still have any issue related to user creation.
